.travis.yml

```diff
@@ -1,22 +1,65 @@
-sudo : false
+sudo: false
 
 notifications:
   email: false
 
 language: C++
 
-compiler: 
-    - gcc
+matrix:
+  include:
+    - compiler: gcc
+      env: 
+        - COMPILE_WITH_MPI=OFF
+        - COMPILER=/usr/bin/g++
+    - compiler: gcc
+      env: 
+        - COMPILE_WITH_MPI=ON
+        - COMPILER=/usr/bin/mpicxx
```

Comment:
On 10/30/2015 08:14 AM, Alberto Sartori wrote:
> only this one gcc + mpi + cxx11=on/off?
It's enough to have only one check without C++11. I chose this one 
because it also runs through the MPI code without C++11, whereas the 
non-MPI compiles skip a whole bunch of code that's #ifdef'd out..